pkgs/shells/marcel/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, python3
+, bash
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "marcel";
+  version = "0.20.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "geophile";
+    repo = "marcel";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DlpjE0RTn2zdTkSKXGvuZXBk8R6ZKrhRe8U7xchyauo=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    dill
+    psutil
+    bash
+  ];
+
+  doCheck = false; # cause the tests provided by marcel will read/write $HOME/.local/share/marcel and /tmp and use sudo
+
+  postFixup = ''
+    substituteInPlace $out/bin/marcel \
+        --replace python3 ${python3}/bin/python3
+    wrapProgram $out/bin/marcel \
```

Comment:
Have you ever tested this?

Reply:
This was the original script:
https://github.com/geophile/marcel/blob/master/bin/marcel
it's just a `bash` script which calls `python3`

This is why the `--prefix PYTHONPATH : "$program_PYTHONPATH"` hack needed to be used:  the default python wrapping logic would never wrap bash scripts

Reply:
I have tested marcel built by this script on my PC, and it works well. Any suggestions about this?